CSIRO/CARS/incoming_handler.sh

```diff
@@ -15,7 +15,7 @@ regex_filter() {
 # $1 - file to handle
 main() {
     local file=$1; shift
-    local regex="^CARS[[:digit:]]{4}_(World_monthly|Australia_weekly)\\.nc$"
+    local regex="^CARS[[:digit:]]{4}_(World_monthly|Australia_weekly).*\\.nc$"
```

Comment:
@smancini told me we would keep the original monthly file on S3/THREDDS but it won't be harvested. I don't think the regex above would achieve that. Also, only the monthly product is targeted for this.

I would go for something like:
```BASH
local regex="^CARS2009_(World_monthly_[[:digit:]]{2}|Australia_weekly)\\.nc$"
```
If that makes sense. By the way, you can note that I removed the possibility to harvest a file named CARSxxxx, this was not necessary.

File names are as follow:
```
CARS2009_World_monthly_01.nc
CARS2009_World_monthly_02.nc
CARS2009_World_monthly_03.nc
CARS2009_World_monthly_04.nc
CARS2009_World_monthly_05.nc
CARS2009_World_monthly_06.nc
CARS2009_World_monthly_07.nc
CARS2009_World_monthly_08.nc
CARS2009_World_monthly_09.nc
CARS2009_World_monthly_10.nc
CARS2009_World_monthly_11.nc
CARS2009_World_monthly_12.nc
```